drivers/hud_apr/app/models/hud_apr/generators/shared/fy2020/question_twenty_six.rb

```diff
@@ -34,8 +34,11 @@ class QuestionTwentySix < Base
           household_ids = relevant_clients.where(population_clause).
             where(ch_clause).
             distinct.pluck(a_t[:household_id])
-          # ignore previously counted households
-          household_ids -= households.to_a
+          # ignore previously counted households, except for the last line (total)
+          if row_index < ch_categories.size - 1
+            household_ids -= households.to_a
+            households += household_ids
```

Comment:
do we ensure household_ids are never `nil`? Seems like we might run into issues with this.